### PR TITLE
Review Survey-microverse

### PR DESCRIPTION
**Pull request by 1GORDON**
# In this project I did the following

* I updated my README file
* I set up my linter files
* I set up my HTML and CSS files

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            <input type="checkbox" id="vessel2" name="vessel2" value="Glass">
+            <label for="vehicle2">Drinking from a Glass</label><br>
+            <br>
+
+            <textarea id="comment" name="comment">Enter your comment here...</textarea><br>
+
+            <br><input type="submit" value="Submit">
+          </fieldset>    
+      </form> 
+
```
Comment (1GORDON) [inline index.html]: Please remove any additional spaces.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+            <label for="cold">Cold</label>
+            <br><br>
+
+            <label for="vessel">Prefered vessel:</label><br>
+            <input type="checkbox" id="vessel1" name="vessel1" value="Bottle">
+            <label for="vessel1">Drinking from a Bottle</label><br>
+            <input type="checkbox" id="vessel2" name="vessel2" value="Glass">
+            <label for="vehicle2">Drinking from a Glass</label><br>
+            <br>
+
```
Comment (1GORDON) [inline index.html]: Please try to remove these blank spaces

**Review by 1GORDON — COMMENTED** [review]
Please make all the necessary changes. : recycle:

File: index.html (hunk 1 above)
Comment (vichuge) [reply index.html]: - [x] Remember is important to include an explanation about WHY is that important to fix, please add the explanation on your comment. 🐶

File: index.html (hunk 2 above)
Comment (vichuge) [reply index.html]: - [x] Remember is important to include an explanation about WHY is that important to fix, please add the explanation on your comment. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments after your inline comments. ⬆️ 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (1GORDON) [reply index.html]: Thank you for the review. let me get to it

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hi @1GORDON 😃, 

- [x] Thanks for working on the required changes. However, you still should update your review's comments with the requested changes applied on them, and not to the commit messages

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — COMMENTED** [review]
Please make the changes suggested. Thank you

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @1GORDON 😃, 

- [ ] **OPTIONAL**
_"Remove blank spaces from html file for good coding practices"_
This was your commit message after the previous reviewer requested you to provide an explanation (a 'why') to your review's comments, and you did it right but it wasn't in the right place. So, I highly recommend that you should pay more attention to that matter. 😄 🙏 

Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**